config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
does this mean we have Redis available now?

Reply:
Rails 5.2 includes an option for a  Redis cache store.  See https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore

We do not have this turned on.  Our default is still the mem cache.